modules/gql/gqlgen/utils.go

```diff
@@ -22,3 +26,18 @@ func Paginate(offset *int, limit *int) (int, int, error) {
 	}
 	return offset_result, limit_result, nil
 }
+
+// Parse optional height, falling back to last processed block if not specified
+func ParseHeight(hb hive_blocks.HiveBlocks, height *model.Uint64) (uint64, error) {
+	var blockHeight uint64
+	if height != nil {
+		blockHeight = uint64(*height)
+	} else {
+		head, headErr := hb.GetLastProcessedBlock()
+		if headErr != nil {
+			return 0, headErr
+		}
+		blockHeight = head
```

Comment:
Getting the last processed block isn't needed. Using max uint64 as the height, will get the latest info.